pom.xml

```diff
@@ -26,6 +26,54 @@
 
     <description>Eclipse Yasson. Reference implementation of JSR-367 (JSON-B).</description>
     <url>https://projects.eclipse.org/projects/ee4j.yasson</url>
+    
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jakarta.json.version>1.1.6</jakarta.json.version>
+        <netbeans.hint.jdkPlatform>JDK_9</netbeans.hint.jdkPlatform>
+    </properties>
+    
+    <dependencies>
+        <!-- Compile dependencies -->
+        <dependency>
+            <groupId>jakarta.json.bind</groupId>
+            <artifactId>jakarta.json.bind-api</artifactId>
+            <version>1.0.2</version>
```

Comment:
Why not to use property as before?

Reply:
similar to above, since the property was only ever used in 1 place, there isn't really a point in having a property. If it was used in 2 or more places then it would make sense to use a property

Reply:
I'll swap it back to the way it was originally (with a property) since this one is just a preference vs. a preference